stock_lot_production_date/models/stock_lot.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class StockProductionLot(models.Model):
+    _inherit = "stock.lot"
+
+    production_date = fields.Datetime(
+        help="This is the date when the goods with this "
+        "lot/serial number have been produced."
```

Comment:
```suggestion
        help=("This is the date when the goods with this "
        "lot/serial number have been produced.")
```

Non blocking suggestion: if you add parenthesis like this, ruff will emit something nicer.

Reply:
@sbidoul Made change as mentioned. Please take a look at it.